README.md

```diff
@@ -69,7 +69,7 @@ object Metrics extends QueryBundle {
     title = s"Number of parameters larger than $n",
     description =
       s"This query identifies functions with more than $n formal parameters",
-    score = 2.0, { cpg =>
+    score = 2.0, f = { cpg =>
```

Comment:
can we call it something more meaningful rather than `f`? 🤔 Maybe `query` ?

Reply:
Yepp. Good idea @tuxology